io/include/pcl/io/hdl_grabber.h

```diff
@@ -48,6 +48,11 @@
 #include <string>
 
 #define HDL_Grabber_toRadians(x) ((x) * M_PI / 180.0)
+#define HDL_DATA_PORT 2368
```

Comment:
This was done because the new VLP_Grabber extends HDL_Grabber.  Rather than making it a public or a protected friend, I opted to make it a DEFINE.